string  ally1_color
string  ally2_color
string  enemy1_color
string  enemy2_color
string  field_pos
int32   ally_robot_count
int32   enemy_robot_count
bool    show_video
